live-examples/js-examples/statement-block.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">var x = 1;
+
+if (x === 1) {
+  var x = 2;
+}
+
+console.log(x);
+// expected output: 2
+
+// Note using let or const instead of var
+// would result in 1 being outputted.
```

Comment:
Hmm, consider illustrating that?

```
var x = 1;
let y = 1;

if (true) {
  var x = 2;
  let y = 2;
}

console.log(x);
// expected output: 2

console.log(y);
// expected output: 1
```

Reply:
Nice example. Will definitely switch! :)